components/support/nimbus-fml/src/main.rs

```diff
@@ -47,6 +47,10 @@ fn main() -> Result<()> {
                         .value_of("package")
                         .map(str::to_string)
                         .or(config.package_name),
+                    channel: matches
```

Comment:
Hmm should the channel be in the `config` or in the `Generate**Cmd`... not so sure